packages/asset-swapper/test/market_operation_utils_test.ts

```diff
@@ -745,9 +739,9 @@ describe('MarketOperationUtils tests', () => {
                 expect(improvedOrders).to.be.length(3);
                 const orderFillSources = improvedOrders.map(o => o.fills.map(f => f.source));
                 expect(orderFillSources).to.deep.eq([
-                    [ERC20BridgeSource.Uniswap],
+                    [ERC20BridgeSource.Uniswap, ERC20BridgeSource.Curve],
                     [ERC20BridgeSource.Native],
-                    [ERC20BridgeSource.Eth2Dai, ERC20BridgeSource.Curve],
+                    [ERC20BridgeSource.Eth2Dai],
```

Comment:
wait what's going on here?

Reply:
Yeah it's a little funky that Curve comes right after Uniswap now, but both ways are technically valid so I'm like w/e :shrug:

Reply:
sus but ok

Reply:
welcome to asset-swapper lol